src/components/Posts/Comments/DisplayComment.vue

```diff
@@ -0,0 +1,189 @@
+        >
+          <q-tooltip anchor="bottom middle" self="center middle">Like</q-tooltip>
+        </q-btn>
+        <q-btn
+          :data-test="'dislike' + comment.text"
+          flat
+          :icon="dislikeIconClass(comment) ? 'img:/icons/thumbs-down-bolder.svg' : 'img:/icons/thumbs-down.svg'"
+          :label="comment.dislikes?.length || 0"
+          rounded
+          size="0.75rem"
+          @click="dislikeComment(comment.id)"
+        >
+          <q-tooltip anchor="bottom middle" self="center middle">Dislike</q-tooltip>
+        </q-btn>
+        <q-btn
+          :data-test="comment.text + '-add-reply'"
+          flat
+          icon="img:/icons/arrow-reply.svg"
+          :label="replyCounter(comment.id)"
+          rounded
+          size="0.75rem"
+          @click="commentStore.setReplyTo(comment.id)"
+        >
+          <q-tooltip anchor="bottom middle" self="center middle">Reply</q-tooltip>
+        </q-btn>
+      </div>
+      <DisplayComment
+        v-for="item of commentStore.getCommentChildren(comment.id)"
+        :document-id="documentId"
+        :collection-name="collectionName"
+        :key="item.id"
+        :comment="item"
+      ></DisplayComment>
+    </div>
+  </q-list>
+</template>
+
+<script setup>
+import { computed, onMounted, ref } from 'vue'
+import { useRouter } from 'vue-router'
+import { shortMonthDayTime } from 'src/utils/date'
+import { useCommentStore, useErrorStore, useUserStore } from 'src/stores'
+import { useQuasar } from 'quasar'
+
+const router = useRouter()
+const userStore = useUserStore()
+const commentStore = useCommentStore()
+const errorStore = useErrorStore()
+
+const props = defineProps({
+  comment: { type: Object, required: true },
+  collectionName: { type: String, required: true },
+  documentId: { type: String, required: true }
+})
+
+const userId = ref('')
+const inputEdit = ref('')
+const isEditing = ref(false)
+const newComment = ref(props.comment.text)
+
+const $q = useQuasar()
+onMounted(async () => {
+  await userStore.fetchUserIp()
+  userId.value = userStore.isAuthenticated ? userStore.getUserRef?.id : userStore.getUserIpHash
+
+  window.addEventListener('keydown', handleKeydown)
+})
+
+async function editComment(commentId, editedComment) {
+  await commentStore
+    .editComment(props.collectionName, props.documentId, commentId, editedComment, userId.value)
+    .then(() => $q.notify({ type: 'info', message: 'Comment successfully edited!' }))
+    .catch((error) => errorStore.throwError(error, 'Failed to edit comment'))
+    .finally(() => (isEditing.value = false))
+}
+
+async function deleteComment(commentId) {
+  await commentStore
+    .deleteComment(props.collectionName, props.documentId, commentId)
+    .then(() => $q.notify({ type: 'positive', message: 'Comment successfully deleted' }))
+    .catch((error) => errorStore.throwError(error, 'Failed to delete comment'))
+
+  // TODO Set all child comments parentId into the current comment Parent Id)
```

Comment:
Are you still going to do this or is it already done?

Reply:
It's not done, now we need to discuss how the comment will work. Then we can decide if we will do it or not

Reply:
Ok, let's discuss tomorrow in the meeting